config/secrets.py.example

```diff
@@ -24,6 +24,19 @@ elif LOCALHOST:
     UPLOADS_DEFAULT_DEST = BASE_DIR + '/uploads/'
     UPLOADS_DEFAULT_URL = 'http://localhost:5000/uploads/'
 
+
+# Database backup path. PRODUCTION and STAGING can be ignored for
+# local testing.
+if PRODUCTION:
+    BACKUP_DEFAULT_DEST = ''
+    BACKUP_DEFAULT_URL = ''
+elif STAGING:
+    BACKUP_DEFAULT_DEST = ''
+    BACKUP_DEFAULT_URL = ''
+elif LOCALHOST:
```

Comment:
What about:

```
if LOCALHOST:
    BACKUP_DEFAULT_URL = 'http://localhost:5000/backup/'
else:
    BACKUP_DEFAULT_URL = ''
```

That way staging and prod are as close as possible (and to minimize the amount of configuration here).